src/domain/replacer.test.ts

```diff
@@ -41,7 +41,8 @@ this is a book`;
 is a pen.`;
     const expected = `following are samples
 
-• one is a pen; and
+• one is a pen;
+and
 
 • one is a pen.`;
```

Comment:
In this pattern, our changes make the results a bit odd, but I think the changes to be more beneficial

```
• one is a pen; and
• one is a pen
```